layers/+lang/bibtex/packages.el

```diff
@@ -110,3 +112,48 @@
         "ld" 'doi-utils-add-bibtex-entry-from-doi
         "li" 'isbn-to-bibtex
         "lp" 'pubmed-insert-bibtex-from-pmid))))
+
+(defun bibtex/init-ebib ()
+  (use-package ebib
+    :defer t
+    :init
+    (spacemacs/set-leader-keys "oe" 'ebib)
```

Comment:
I think the keybinding for this should be changed since "o" is a reserved binding for users, as laid out in [CONVENTIONS](https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#user-prefix). My suggestion would be to use "ate" since `ebib` is a tool, although "ae" is not a bad idea too since I don't remember seeing any layers with a binding to it.

Reply:
Ah yes, good point. Indeed, this was my personal, private layer, keybinding. The keybinding "ae" [is reserved for "email" applications](https://github.com/syl20bnr/spacemacs/blob/64fed6f7daba47144b09403a4982f26eb79b8ec6/layers/%2Bspacemacs/spacemacs-defaults/keybindings.el#L36).

Reply:
I see! Then the "ate" I suggested earlier might be the binding to go then.